core_data_modules/traced_data/io.py

```diff
@@ -439,6 +445,305 @@ def import_coda_to_traced_data_iterable_as_matrix(cls, user, data, key_of_raw, c
             td.append_data(td_matrix_data, Metadata(user, Metadata.get_call_location(), time.time()))
 
 
+class TracedDataCoda2IO(object):
```

Comment:
This has an unfortunate alternative reading: 'TraceData Coda 2 IO' which confused me at first. Not sure if there's much we can do about that

Reply:
Is `TracedDataCodaV2IO` any better?

Reply:
Yup. Thanks.

Reply:
changed